Fix taken from OOO_STABLE_1 branch
#104564# GetDefaultFont(): illegal assignment of invalid pointer corrected

Index: oo_cvs/vcl/source/gdi/outdev3.cxx
===================================================================
RCS file: /cvs/gsl/vcl/source/gdi/outdev3.cxx,v
retrieving revision 1.74.2.2
retrieving revision 1.74.2.3
diff -u -3 -p -r1.74.2.2 -r1.74.2.3
--- oo_cvs/vcl/source/gdi/outdev3.cxx	24 May 2002 14:10:47 -0000	1.74.2.2
+++ oo_cvs/vcl/source/gdi/outdev3.cxx	31 Oct 2002 20:27:17 -0000	1.74.2.3
@@ -2067,7 +2067,7 @@ Font OutputDevice::GetDefaultFont( USHOR
             if ( pSearch3 )
             {
                 aNames += ';';
-                aNames.AppendAscii( pSearch2 );
+                aNames.AppendAscii( pSearch3 );
             }
 
             // Search Font in the FontList
